metadata-ingestion/src/datahub/emitter/mcp_builder.py

```diff
@@ -167,6 +171,9 @@ def gen_containers(
         aspectName="dataPlatformInstance",
         aspect=DataPlatformInstance(
             platform=f"{make_data_platform_urn(container_key.platform)}",
+            instance=f"{make_dataplatform_instance_urn(container_key.platform, container_key.instance)}"
```

Comment:
I think we should instead add a new argument to the function itself and use that - I recall there is some inconsistency among sources in terms of how the "instance" field is populated :( 

cc @treff7es To take a look!

Reply:
I know in some cases _environment_ values (dev, ei, prod, ...) are used for the _platform instance_. This was decided to keep backward compatibility with an issue that was introduced previously in some connector and so to avoid migrating URNs. Is that what you mean?

Anyway, not clear for me if this is a comment blocking this PR or just an observation for a future improvement. If there is something specific I could do in this PR, just let me know please.